src/uwtools/tests/config/test_jinja2.py

```diff
@@ -287,11 +287,20 @@ def test_unrendered(s, status):
     assert jinja2.unrendered(s) is status
 
 
-@mark.parametrize("tag", ["!bool", "!datetime", "!float", "!int"])
-def test__deref_convert_no(caplog, tag):
+@mark.parametrize(
+    "tag,value",
+    [
+        ("!datetime", "foo"),
+        ("!dict", "foo"),
+        ("!float", "foo"),
+        ("!int", "foo"),
+        ("!list", "null"),
+    ],
```

Comment:
`foo` _is_ an acceptable value using the `!list` tag because it gets converted into the list `['f', 'o', 'o']`, so I used `@mark.parametrize` to include an alternative bad case. The `!bool` tag was removed from this test because all objects in Python can be converted into `bool` values.